app/scripts/badges.js

```diff
@@ -12,14 +12,14 @@ export default () => [
 
 
 '<div style="width: 100px; height: 100px; display: inline; padding-right: 5px" id="my-badge" class="tooltip">',
-'<span class="tooltiptext">Percentage of Repository Activity</span>',
+'<span class="tooltiptext">Percentage calculated based on the number of commits, the number of contributors, accepted PRs, and active issues of the last month. Also, if release notes exist.</span>',
```

Comment:
About PRs, is not only acceptance that affects the total Score. Everything that can happen in PRs is considered.
The release note not only must exist but must be recent.